src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/Editor/DefaultTextBufferProjectService.cs

```diff
@@ -81,7 +81,7 @@ public override string GetProjectPath(object project)
         }
 
         // VisualStudio for Mac only supports ASP.NET Core Razor.
-        public override bool IsSupportedProject(object project) => true;
+        public override bool IsSupportedProject(object project) => project is DotNetProject;
```

Comment:
Made this a little more restrictive since technically there's a lot of project types in VS for Mac and we only understand one of them 😄.

Reply:
OK, sounds good